packages/code-du-travail-api/src/server/routes/idcc/idcc.elastic.js

```diff
@@ -18,7 +18,7 @@ function getIdccBody({ query }) {
                 match: {
                   "shortTitle.french": {
                     query: `${query}`,
-                    fuzziness: "AUTO",
+                    fuzziness: "1",
```

Comment:
This is to prevent `invest` from matching with `industrie` without completely removing fuzziness

Reply:
do you try with "2"

Reply:
I could not see the difference with "AUTO" when setting it to "2". `invest` would still match `industrie`